MapifyEditor/Export/Validators/GameContent/StationValidator.cs

```diff
@@ -26,8 +27,16 @@ protected override IEnumerator<Result> Validate(Scenes scenes)
 
                 if (string.IsNullOrWhiteSpace(station.stationName))
                     yield return Result.Error($"Station '{station.name}' must have a name", station);
+
                 if (string.IsNullOrWhiteSpace(station.stationID))
                     yield return Result.Error($"Station '{station.name}' must have an ID", station);
+
+                var alphaNumericRegex = new Regex(@"^[a-zA-Z0-9]*$");
```

Comment:
You should be able to use the static Regex.IsMatch method here. If not we should statically cache it in the class.

Reply:
changed